package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "resend",
-  "version": "4.3.0",
+  "version": "4.4.0-canary-0",
```

Comment:
Can you revert that, we can bump the version once we release since it may include more stuff